bin/deploy-docker.sh

```diff
@@ -4,6 +4,9 @@
 # Environment variables passed to this script (eg IMAGE_TAG) will be available to the given docker-compose.yaml file
 
 cmdname="$(basename "$0")"
+bin_path="$(cd "$(dirname "$0")" && pwd)"
```

Comment:
At one point, you suggested killing the `docker` subdir and moving docker files to the root of the project.  Would this be a cleaner approach, now that we're effectively docker committed?

Reply:
>At one point, you suggested killing the docker subdir and moving docker files to the root of the project

Some continuous integration services expect a Dockerfile at repo root (Heroku), but we aren't using any at the moment, and that would be the only reason.

>killing the `docker` subdir

I actually prefer a separate `docker/` directory for the docker-specific functionality :). The root of a repo can tend to get crowded with config/dotfiles

>now that we're effectively docker committed

I don't want to give it labels :wink: , but I've tried to make our tooling as free-standing as possible (ie useable independently outside our docker deployment method).